cfme/services/catalogs/catalog.py

```diff
@@ -137,41 +126,65 @@ def exists(self):
             return False
 
 
-@navigator.register(Catalog, 'All')
+@attr.s
+class CatalogCollection(BaseCollection):
+    """A collection for the :py:class:`cfme.services.catalogs.catalog.Catalog`"""
+    ENTITY = Catalog
+
+    def create(self, name, description, items=None):
+        """Create a catalog.
+
+        Args:
+            name: The name of the catalog
+            description: The description of the catalog
+            items: Items in the catalog
+        """
+        view = navigate_to(self, 'Add')
+        view.fill({
+            'name': name,
+            'description': description,
+            'assign_catalog_items': items
+        })
+        view.add_button.click()
+        catalog = self.instantiate(name=name, description=description, items=items)
+        view = self.create_view(CatalogsView)
+        view.flash.assert_success_message('Catalog "{}" was saved'.format(name))
```

Comment:
**Required**

A bit of scope creep, but I'd like to require this because its so small.

Please move the flash message validation to the test_catalog_crud test that's already being updated below, and only use `view.flash.assert_no_error` here.

This keeps a changing flash message from blocking/breaking tests that create a catalog item as a setup step.